pkg/server/events.go

```diff
@@ -316,6 +327,12 @@ func handleContainerExit(ctx context.Context, e *eventtypes.TaskExit, cntr conta
 			return errors.Wrapf(err, "failed to load task for container")
 		}
 	} else {
+		if nric != nil {
+			if _, err := nric.InvokeWithSandbox(ctx, task, v1.Delete, nriSB); err != nil {
+				log.G(ctx).WithError(err).Errorf("Failed to delete nri for %q", cntr.ID)
+			}
+		}
+
```

Comment:
instead of these code blocks what do you think of adding
```
containerd.With_NRI_Invoke_SB(
   &nri.Sandbox{ID: cntr.SandboxID,}..., 
```
to task.Delete?

Reply:
ya, makes sense, I can add some opts to handle this more cleanly